### PR TITLE
1773 by Inlead: Improve holdings display.

### PR DESCRIPTION
**Pull request by NicolaeGrigoriu**
http://platform.dandigbib.org/issues/2048

### CODE REVIEW

File: modules/fbs/includes/fbs.availability.inc [hunk 1]
```diff
@@ -66,7 +66,7 @@ function fbs_availability_holdings($provider_ids) {
       'is_periodical' => FALSE,
     );
 
-    $placement_parts = array('branch', 'department', 'location', 'sublocation');
+    $placement_parts = array('branch', 'department', 'location', 'sublocation', 'materials');
```
Comment (guddo) [inline modules/fbs/includes/fbs.availability.inc]: If the line declaring an array spans longer than 80 characters, each element should be broken into its own line.

File: modules/fbs/fbs.module [hunk 2]
```diff
@@ -382,6 +382,34 @@ function fbs_form_ding_reservation_reservations_form_alter(&$form, &$form_state)
   }
 }
 
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * FBS API delivers 1 more holding level, called "MaterialgroupDescription".
+ * A new option is added in order to show or not this.
```
Comment (guddo) [inline modules/fbs/fbs.module]: .."_to show this or not._"*

File: modules/fbs/fbs.module [hunk 3]
```diff
@@ -382,6 +382,34 @@ function fbs_form_ding_reservation_reservations_form_alter(&$form, &$form_state)
...
+    '#default_value' => variable_get('show_material_group', 0),
+  );
+  $form['holdings']['show_dk5_classification'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Show dk5 classification'),
+    '#default_value' => variable_get('show_dk5_classification', 0),
+  );
+  $form['holdings']['show_sorted_creator'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Show sorted author'),
```
Comment (guddo) [inline modules/fbs/fbs.module]: sorted = inverted*

**Comment by cableman** [comment]
Reviewed and approved

**Pull request merged**